clients/python/pycellbase/__init__.py

```diff
@@ -1 +1 @@
-__version__ = '4.5.3'
+__version__ = '4.5.4'
```

Comment:
We try to keep the major and minor version numbers in sync with the main package. Develop is supposed to be 4.7.0-SNAPSHOT, so we should update the version of the Python client to 4.7.0